pom.xml

```diff
@@ -33,6 +33,11 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>branch-api</artifactId>
```

Comment:
It is the second PR (from two different developers) that you rejected by the same reason. Maybe something is wrong here.